source/use-cases/sensitive-data-encryption.txt

```diff
@@ -124,7 +124,153 @@ In this step, we generate a local master key to expedite setup of our developmen
           }
 
 
-B. Define a JSON Schema
+B. Create a Data Encryption Key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this section, we use the MongoDB driver to create and store a data
+encryption key, used by CSFLE to automatically encrypt and decrypt data on
+document fields. The following configuration values are required by the client
+to generate a new data encryption key:
+
+* The local master key or AWS KMS master key access settings.
+* A MongoDB connection string that authenticates on a running server.
+* The key vault database and collection name.
+
+Upon creating a data encryption key, the driver returns the ``_id`` field
+(as a `UUID <https://en.wikipedia.org/wiki/Universally_unique_identifier>`_)
+of a document that stores information about the key including:
+
+* The data encryption key, encrypted by the master key.
+* The key management service provider data.
+
+Follow the steps below to generate a single data encryption key from the
+local master key.
+
+.. note::
+
+  If using AWS KMS to provide the master key, refer to the "`F. Convert to
+  a Remote Master Key`_" section for the appropriate settings.
```

Comment:
nit: the "F" in the step title is a bit weird. Instead we could add an inline reference above the step heading and then use a `:ref:` here instead

**Inline reference above the step heading**
```
.. _fle-convert-to-a-remote-master-key:

F. Convert to a Remote Master Key
```

**Reference in this note**
```
If using AWS KMS to provide the master key, refer to the :ref:`Convert
to a Remote Master Key <fle-convert-to-a-remote-master-key>` section of
this guide for the appropriate settings.
```

Reply:
Thanks, I was looking for that directive, but couldn't find it for some reason. Will fix.